Wrapping/R/CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ set_source_files_properties(${swig_generated_file_fullname} PROPERTIES COMPILE_F
 # set the output directory for the R library to the binary packaging location
 set_target_properties( ${SWIG_MODULE_SimpleITK_TARGET_NAME} PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/Packaging/SimpleITK/src/)
 
+if (WIN32)
+	set_target_properties( ${SWIG_MODULE_SimpleITK_TARGET_NAME} PROPERTIES LINK_FLAGS "-s -shared -static-libgcc -static-libstdc++ -static -lpthread" )
+endif()
```

Comment:
I went around in circles getting this to work. Here's what I remember.
Setting CMAKE_*_LINKER_FLAGS didn't seem to do the job, but I'm not sure why. That was what I tried first. I think they may have broken other tests.

the -static flag gets set by the normal package build process (which we aren't using), and I think the normal version of mingw in the RTools package must have different defaults set up causing the appropriate libraries to be linked. If I don't have the std libraries linked statically I get missing symbols.

I wasn't happy with the pthread situation either. There doesn't seem to be any standard way of dealing with this under R/windows that I can see.